hazelcast/src/main/java/com/hazelcast/client/impl/protocol/codec/RingbufferAddAllCodec.java

```diff
@@ -104,7 +101,7 @@ public static ClientMessage encodeRequest(java.lang.String name, java.util.Colle
     public static class ResponseParameters {
 
         /**
-         * the ICompletableFuture to synchronize on completion.
+         * the CompletionStage to synchronize on completion.
```

Comment:
These codecs are auto-generated and we dont allow them to be changed by hand. You need to update [this](https://github.com/hazelcast/hazelcast-client-protocol/blob/master/protocol-definitions/Ringbuffer.yaml#L250) protocol definition, send a pull request to protocol repo and regenerate the codecs as described in the README. (`python3 generator.py -r /your/hazelcast/repo/path`)

Reply:
I submitted the [PR](https://github.com/hazelcast/hazelcast-client-protocol/pull/253) and pushed generated codec.
Thanks for help.